confgenerator/confgenerator.go

```diff
@@ -157,7 +157,7 @@ func (l *Logging) generateFluentbitComponents(userAgent string, hostInfo *host.I
 					components = append(components, processor.Components(tag, strconv.Itoa(i))...)
 				}
 				components = append(components, setLogNameComponents(tag, rID)...)
-				logNames = append(logNames, regexp.QuoteMeta(rID))
+				logNames = append(logNames, regexp.QuoteMeta(tag))
```

Comment:
This is no longer `logNames`, is it? Want to call it `tags` or `tagRegexps` instead?

Reply:
Changed to `tags`.